class.jetpack.php

```diff
@@ -1319,6 +1321,20 @@ public static function show_development_mode_notice() {
 		}
 	}
 
+	public static function show_sync_lag_notice() {
+		if ( ! class_exists( 'Jetpack_Sync_Queue' ) ) {
```

Comment:
since `require_once` is so cheap to execute, you probably don't need the enclosing if-statement.